tests/functional/schema/good_data/one_lookup_results.json.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Ansible Project
+SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
This and some of the other `.license` files don't have both types of license identifier comments.

Reply:
Thanks, fixed in 52a67d7a92309bdd7358d940171265b5fea33310.